README.md

```diff
@@ -12,6 +12,40 @@ https://github.com/ExpediaGroup/incubator-iceberg/tree/build-hiveberg-modules an
 ```
 Ultimately we would like to contribute this Hive Input Format to Iceberg so this would no longer be required.
 
+## Features
+### IcebergInputFormat
+
+To use the `IcebergInputFormat` you need to write a Hive query similar to:
+```sql
+CREATE TABLE source_db.table_a
+   ROW FORMAT SERDE 'com.expediagroup.hiveberg.IcebergSerDe'
+   STORED AS
+        INPUTFORMAT 'com.expediagroup.hiveberg.IcebergInputFormat'
+        OUTPUTFORMAT 'org.apache.hadoop.hive.ql.io.HiveIgnoreKeyTextOutputFormat'
+   LOCATION 'path_to_iceberg_data_warehouse';
+```
+
+You must ensure that:
+- You are creating Iceberg tables using the Iceberg `HadoopCatalog` API. 
+    - Using this means all your Iceberg tables are created under one common location which you must point the Hive table `LOCATION` at. 
+- Ensure your Hive table has the **same** name as your Iceberg table. 
+
+For example, if you created the Hive table as shown above then your Iceberg table must be created using a `TableIdentifier` as follows where both table names match: 
+```
+TableIdentifier id = TableIdentifier.parse("source_db.table_a");
+```
+
+### IcebergStorageHandler
+This is implemented as a simplified option for creating Hiveberg tables. The Hive query should instead look like:
+```sql
+CREATE TABLE source_db.table_a
+   STORED BY 'com.expediagroup.hiveberg.IcebergStorageHandler'
+   LOCATION 'path_to_iceberg_data_warehouse';
+```
+
+### Predicate Pushdown
+Pushdown of the HiveSQL `WHERE` clause has been implemented so that filter's are pushed to the Iceberg `TableScan` level as well as the Parquet `Reader`. ORC implementations are still in the works. 
```

Comment:
```suggestion
Pushdown of the HiveSQL `WHERE` clause has been implemented so that filters are pushed to the Iceberg `TableScan` level as well as the Parquet `Reader`. ORC implementations are still in the works. 
```